<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tabs</class>
 <widget class="QMainWindow" name="Tabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Groceries</string>
  </property>
  <widget class="QWidget" name="top">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="recipesTab">
       <attribute name="title">
        <string>Recipes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="recipesView"/>
        </item>
        <item>
         <widget class="QWidget" name="frame">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="bAddRecipe">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bDeleteRecipe">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="groceriesTab">
       <attribute name="title">
        <string>Groceries</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Planned Recipes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="placeholderText">
              <string>Add: &lt;name&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="plannedView"/>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="bDeletePlanned">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bClearPlanned">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Generated List</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="leGrocery">
             <property name="placeholderText">
              <string>Add: &lt;quantity&gt; [&lt;unit&gt;] &lt;name&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="groceriesView"/>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="bResetGroceries">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bClearGroceries">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="foodsTab">
       <attribute name="title">
        <string>Foods</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="leFood">
          <property name="placeholderText">
           <string>Add: &lt;name&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="foodsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="recipeTab">
       <attribute name="title">
        <string>Recipe</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="editRecipeControls">
          <property name="title">
           <string>Edit</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="leRecipeTitle">
             <property name="placeholderText">
              <string>Title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leRecipeStep">
             <property name="placeholderText">
              <string>Add Ingredient: &lt;quantity&gt; [&lt;unit&gt;] &lt;name&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leRecipeIngredient">
             <property name="placeholderText">
              <string>Add Step</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="editRecipeControlsSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bSubmitRecipe">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bClearRecipe">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCancelRecipe">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="recipeViewContainerLayout">
          <item>
           <widget class="QTableView" name="recipeView"/>
          </item>
          <item>
           <widget class="QPushButton" name="bEditRecipe">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
